src/redux/auth/auth.actions.ts

```diff
@@ -45,8 +45,7 @@ export const userLogout = createAsyncThunk(
       });
       return res;
     } catch (error) {
-      console.log(error);
-      throw error;
+      throw { message: "Failed to log in", originalError: error };
```

Comment:
This is not the correct syntax for throwing errors

Reply:
changed to: throw new Error("Failed to logout. Please try again");